### ISSUE TITLE
[dev.icinga.com #9026] Add To Dashboard ... on the dashboard

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1593]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/9026

**Created by jmeyer on 2015-04-09 08:10:27 +00:00**

Assignee: _aklimov_
Status: _Resolved_  (closed on _2015-04-30 15:55:04 +00:00_)
Target Version: _2.0.0-rc1_
Last Update: _2015-04-30 15:55:04 +00:00 (in Redmine)_

---

This should imho read "Add New Pane Or Dashlet" (or whatever our naming is/should be) as "Add To Dashboard" means to me that I am able to add the current Pane to itself or a new one.

**Changesets**

_2015-04-30 15:47:23 +00:00_ by _aklimov_ ffe23a184f0a1659ffeb8cfc0d05a7b36fd9a7c1

    Don't display "Add To Dashboard" on the dashboard
    
    resolves #9026
